File: 04_1차원_배열/Problem_03.cpp
/* 04단계 [ 1차원 배열 ] */
// 제목 : 최소, 최대
// 문제 번호 :10818

#include "iostream"
using namespace std;

int main(void)
{
    int n;
    int min = 1000000;
    int max = -1000000;

    cin >> n;
    int array[n];

    for (int i = 0; i < n; i++)
    {
        cin >> array[i];
        if (array[i] < min)
            min = array[i];
        if (array[i] > max)
            max = array[i];
    }
    cout << min << " " << max << endl;
}

File: Python/04_1차원_배열/Problem_03.py
min = 1000000
max = -1000000

n = int(input())
array = list(map(int, input().split()))

for i in range(n):
    if array[i] > max:
        max = array[i]
    if array[i] < min:
        min = array[i]
print(f"{min} {max}")
